File: tests/files/output/python/test1.py
class myClass:
    def create_string(self):
        return 'hello'

    def create_list(self):
        return [1, 2, 3]
my_instance = myClass()
my_string = my_instance.create_string()
my_string += 'append'
my_list = my_instance.create_list()
my_list.append(4)
my_string_len = len(my_string)
my_list_len = len(my_list)

File: tests/files/output/php/test1.php
class myClass {
    public function create_string() {
        return 'hello';
    }

    public function create_list() {
        return [1, 2, 3];
    }
}
$my_instance = new myClass();
$my_string = $my_instance->create_string();
$my_string .= 'append';
$my_list = $my_instance->create_list();
$my_list[] = 4;
$my_string_len = strlen($my_string);
$my_list_len = count($my_list);